cmd/sebak/cmd/node.go

```diff
@@ -280,10 +281,32 @@ func runNode() {
 
 		os.Exit(1)
 	}
-	nr := sebak.NewNodeRunner(flagNetworkID, currentNode, policy, nt, isaac, st)
-	if err := nr.Start(); err != nil {
-		log.Crit("failed to start node", "error", err)
 
+	// Execution group.
+	var g run.Group
+	{
```

Comment:
Just curious, why this brace needs?

Reply:
@Charleslee522 If you cover your code with curly braces, the newly assigned variables in the block will be local variable, that is, it will not be available from outside of the block. Without this feature, it can divide the code block **VISUALLY** :)

Reply:
Also known as [lexical scoping](https://whatis.techtarget.com/definition/lexical-scoping-static-scoping) ;)